docs/learning/WriteAConnector.md

```diff
@@ -80,9 +93,9 @@ See [Section on iTwin Synchronization](#ways-to-sync-data-to-an-itwin) for more
 
 However, in some instances, where a specific format is not covered, one can start to develop a new Connector using the [iTwin.js SDK](https://github.com/iTwin/itwinjs-core)
 
-The imodel-bridge package provided as part of the iTwin.js SDK makes it easier to write an iTwin Connector backend that brings custom data into a digital twin. To run this environment with the iTwin.js library that this package depends on requires a JavaScript engine with es2017 support.
+The ConnectorFramework package provided separately from the iTwin.js SDK makes it easier to write an iTwin Connector backend that brings custom data into a digital twin. To run this environment with the iTwin.js library that this package depends on requires a JavaScript engine with es2017 support.
```

Comment:
I wouldn't change the wording here, the Connector framework is still a very important part of the iTwin.js library. It's just no longer in the iTwin.js Core repository (which mostly works with iModels directly save for some exceptions).

I'd leave `as part of`